examples/gslib/exs.cpp

```diff
@@ -0,0 +1,177 @@
+﻿//          Serial example of utilizing GSLib's FindPoints methods
+//
+// Compile with: make exs
+//
+// Sample runs:
+//    ./exs -m ../../data/rtaylor2D-q3.mesh -o 3
+//    ./exs -m ../../data/fichera.mesh -o 3
+
```

Comment:
Pick a better name, e.g. `findpts` and `pfindpts` and a provide a description about what the example does, e.g. "Demonstrate evaluation of a grid function in a set of points in physical space using GSlib/FindPoints."